### DISCUSSION TITLE
Supporter license

### DISCUSSION THREAD

**Discussion by nomadphase** [brianpetro/obsidian-smart-connections#285] [General]
Hello Brian,

Love your work on this and keen to support it, particularly if the vision is aligned to my values and needs.
May I ask, what additional functionality is provided by the supporter license?
Do you have a roadmap?

Cheers

**Comment by brianpetro** [comment]
Hi @nomadphase 

Thanks for your interest in supporting my work on Smart Connections 😊

In short, having a small, engaged, group of people with whom I can test new features, including ideas that would be too costly or complicated to ship publicly, is the purpose behind the supporter program.

Becoming a supporter has two primary benefits:

1) Priority communications: that includes supporter-only updates and a special email address for supporters to contact me about Smart Connections. In the future, I'd like to expand this further, maybe with a supporter-only forum, but for now it's just via email. 

2) Experimental and early-access to new features: currently, that includes the Smart Connections ChatGPT plugin integration. This is what triggered me to start the supporter program because feature like the ChatGPT plugin have a backend server requirement, so supporters help me recover that cost.

As far as a roadmap that's specific to supporters:

I don't think that a roadmap aligns with my vision for the supporter program. Instead, I envision the supporter program as like my playground where I can explore and try new things, experimenting and gaining feedback necessary to contribute to the overall Smart Connections roadmap. 

For example, as the Smart Connections ChatGPT plugin matures, I'd like to release it as its own product with a lower barrier to entry, like $10/month instead on the $100+/year payment from supporters, and in exchange for early access, the participation of supporters will help me get to the point that the software is ready to be a more clear-cut product like that.

An additional feature, which I wouldn't quite consider part of the roadmap because I don't have an ETA at the moment, that will also be supporter-first is "external integrations" as described in https://github.com/brianpetro/obsidian-smart-connections/issues/27

I hope it's becoming clear that, rather than being a product offering (which I do intend to provide in the future) the supporter program's purpose is to facilitate that growth and development of Smart Connections.

Lastly, if you become a supporter and decide that being part of the program isn't right for you, then I'll happily provide a refund, and hopefully gain some valuable feedback about what didn't meet your expectations in the process :)

Thanks again for your interest in supporting Smart Connections,
Brian 🌴

**Comment by nomadphase** [comment]
Thanks Brian,

I have signed up for a supporter key to contribute to and support your work

**Reply by brianpetro** [reply]
Thank you @nomadphase 😊

Brian 🌴

**Reply by dougkeiller** [reply]
Hi Brian, 

I'm going to sign up. I'm looking forward to interacting with you around this product.

I migrated from Mem.ai to Obsidian, largely because of the Smart Connections plugin. Might even make sense, when you productize, to go after Mem.ai and other Cloud 'second brain' / PKM tools that are worried about startup failures and want local files, but also the easy functionality of the cloud tools.

Obsidian seems like one of those products that's experienced early adoption by tech forward types, but remains both sufficiently complex to scare away non-tech types, and without enough functionality out of the box to quickly win new users.

I wonder if there's a way to bundle Smart Connections, and maybe 1-2 other game-changing plugins, with Obsidian so that the awesome power is immediately clear upon first use?

**Reply by brianpetro** [reply]
Hi @dougkeiller , and thanks for the support!

> I migrated from Mem.ai to Obsidian, largely because of the Smart Connections plugin

That's quite the compliment, thank you 😄

I don't have any plans for offering more than a supporter-type offer, which mainly exists to help me maintain meaningful communication with users while keeping the volume at something manageable. 

That said, I am working on a product, and there may be some synergy there with Smart Connections in the future.

Obsidian does seem to require that users have either a specific need that's unmet by alternatives or a high tolerance to steep learning curves. And even with a tech background, for example software engineers, while they see the utility, putting it into practice seems to often be out of reach for them too. Among my software engineering acquaintances, the most frequent sentiment I hear about Obsidian is "I need to use it more." 

So some things could definitely be improved there. I do know the current position on AI from the Obsidian team is to wait for local models, so not sure if they'd be very interested in this plugins. And though local models for embeddings will be coming in Smart Connections v2, there isn't much on the horizon for chat models, but things are changing fast so idk we'll see.

Thanks for your thoughts and support. I also look forward to hearing any thoughts or insights you may have going forward.

🌴

**Comment by wadib** [comment]
Again the same longing for something missing in my PKM.
How can I become a supporter, can't seem to find it anywhere?

**Reply by brianpetro** [reply]
Hi @wadib 

The "Become a Supporter" button is located in the Smart Connections settings within Obsidian.

🌴

**Reply by donbarger** [reply]
When I click that button, nothing happens. All of the other buttons work except that one.

**Reply by brianpetro** [reply]
@donbarger thanks for bringing that to my attention. Should be fixed in `v2.2.81` 🌴

**Comment by coneos** [comment]
Hi Brian,
Smart connect looks great but the entry ticket is a bit pricy without being able to test it first. Do you know when you can offer a monthly subscription?

**Reply by christianngarcia** [reply]
Thank you for your interest and feedback! I’m happy to share that we do offer a monthly subscription option. You can find all the details and get started here: https://smartconnections.app/smart-connect/
